xml/bp_troubleshooting_dashboard.xml

```diff
@@ -88,16 +88,37 @@
      </step>
      <step>
        <para>
-         Check the &dashboard;mgr log file for any errors:
-<!-- where do I Find this logfile? Tatjana is checking-->
+         Check the &mgr; log file for any errors. In order to check the
+         log file you will need to get the name of the &mgr; daemon by
+         the <command>cephadm ls</command> command first. It will give
```

Comment:
```suggestion
         Check the &mgr; log file for any errors. To do this, first you will need
         the name of the &mgr; daemon. Find this with the <command>
         cephadm ls</command> command. It will return output similar to the
```

Reply:
(re-wrap if you do not like line-wrapping markup. I know some do not. I do not think we have specific guidance on doing it or not.)